package.json

```diff
@@ -131,10 +131,10 @@
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.0",
     "babel-plugin-require-context-hook": "^1.0.0",
-    "backpack-node-sass": "^0.7.1",
+    "backpack-node-sass": "^0.9.0",
```

Comment:
You may need to install a newer version of `node-sass` as this newer version installs minimum v7 which at the same time means we need to bump `sass-loader`

Reply:
will remove it altogether as we're not actually building the css files as part of our release process anymore and no one is using the css packages anyways so we can remove it!